modules/architecture-machine-roles.adoc

```diff
@@ -34,8 +34,8 @@ The `kubelet` service must not be newer than `kube-apiserver`, and can be up to
 |===
 [.small]
 --
-1. For example, {product-title} 4.5, 4.7, 4.9, 4.11.
-2. For example, {product-title} 4.6, 4.8, 4.10.
+1. For example, {product-title} 4.9, 4.11.
```

Comment:
@kalexand-rh I updated to remove unsupported OCP versions here, let me know if you want me to revert that.

Reply:
Thank you for catching that!